framework/wazuh/security.py

```diff
@@ -824,11 +1118,11 @@ def set_role_policy(role_id, policy_ids, position=None):
             policy_id = int(policy_id)
             role_policy = rpm.add_policy_to_role(role_id=role_id[0], policy_id=policy_id, position=position)
             if role_policy == SecurityError.ALREADY_EXIST:
-                result.add_failed_item(id_=policy_id, error=WazuhError(4011))
+                result.add_failed_item(id_=int(policy_id), error=WazuhError(4011))
```

Comment:
Why is it necessary to recast policy_id to int if it was done in L#1118?

Reply:
It was unnecessary. Removed to avoid confusion